content/grants/2023/community-meetups.md

```diff
@@ -0,0 +1,37 @@
+assignee = [""]
+grant_id = "B0165"
+champion = ["~dalweb-donfun"]
+completed = false
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0165&prefill_Grant+Name=Meetup%20Bounty"
++++
+
+# Requirements
+- Meetups should be geographically based.
+- You must host a total of six meetups, with a targeted cadence of once a month.
+- Meetups must be publicized and include a web presence like meetup.com, and should be actively promoted via social media, in-person recruitment, flyers, a website, etc. If you need a signal boost, email thomas@urbit.org.
+- Create and admin an Urbit group for your community to which all meetup attendees will be invited. This group should be actively maintained with info about upcoming events, copies of past presentation materials, and any other related meetup content.
+- Share all the materials you generate in service of your meetups like, presentations, flyers, FAQs or anything else in ~siddef/uf-meetups.
+- Every meetup must be publicized on the [Events](https://urbit.org/events) section on urbit.org via a commit to the urbit.org [github repo](https://github.com/urbit/urbit.org). 
+- Your community should have a home on the [community section](https://urbit.org/ecosystem?type=communities) of our ecosystem page. You can add this to the website with a commit to the urbit.org [github repo](https://github.com/urbit/urbit.org). If you have never submitted a pull request on github before, let us know and we will teach you how.
+- Write a short meetup report about all meetups including: pictures of the meetups, 50-200 words description of every meetup and the attendees statistics of the meetups. At the end of your grant you can send this to thomas@urbit.org for completion of your grant.  
+# Encouraged Features
+- Semi-regular presentations that introduce Urbit to newcomers, help members boot a planet, or introduce the more technical side of urbit are strongly suggested.
+- Having a collection of recommended groups & apps to which you can direct meetup attendees, such that they can begin to explore the Urbit network.
+- As one of the main goals of meetups is to get people building on Urbit, once your meetup gains some traction, consider looking into writing a proposal for a grant on which members of your meetup can collaborate.
+- While the specific requirements of the meetup bounty can be summarized as "run a regular meetup", the overarching goal is to build a local community and onboard people to Urbit. You should become a true champion of your local urbit community who helps people find their own path in urbit, whether it is signing up for a course, a grant or even working for a company in the ecosystem. It's up to you to find the potential in the attendees.
+# Application process
+After applying to this bounty, set-up a call with ~dalweb-donfun who will assess the success of your strategy and run you through the requirements of this bounty. If everything checks-out you are good to go. 
```

Comment:
A strategy can't really be deemed a success before it's been tried. Maybe "the soundness" of your strategy or similar?